app/src/main/java/agency/five/codebase/android/movieapp/ui/moviedetails/MovieDetailsScreen.kt

```diff
@@ -0,0 +1,242 @@
+    modifier: Modifier,
+    movieDetailsViewState: MovieDetailsViewState,
+    onClickFavoriteButton: () -> Unit
+) {
+    Box(
+
+    ) {
+        AsyncImage(
+            model = movieDetailsViewState.imageUrl,
+            contentDescription = movieDetailsViewState.title,
+            contentScale = ContentScale.Crop,
+            modifier = Modifier
+                .height(370.dp)
+        )
+        FavoriteButton(
+            isFavorite = movieDetailsViewState.isFavorite,
+            modifier = Modifier
+                .size(40.dp),
+            onClick = { onClickFavoriteButton() }
+        )
+        Column(
+            modifier = Modifier
+                .align(Alignment.BottomStart)
+                .background(Color.Black.copy(0.7f))
+                .fillMaxWidth()
+        ) {
+            Row(
+                modifier = Modifier
+                    .padding(top = 5.dp)
+            ) {
+                UserScoreProgressBar(
+                    userScoreProgressBarViewState = UserScoreProgressBarViewState(
+                        score = movieDetailsViewState.voteAverage
+                    ),
+                    modifier = Modifier
+                        .padding(start = 10.dp, top = 5.dp)
+                )
+                Text(
+                    text = "User score",
+                    fontSize = 12.sp,
+                    fontWeight = FontWeight.SemiBold,
+                    fontFamily = FontFamily.Serif,
+                    color = Color.White,
+                    modifier = Modifier
+                        .padding(start = 10.dp, top = 17.dp)
+                )
+            }
+            Text(
+                text = movieDetailsViewState.title,
+                fontSize = 20.sp,
+                fontWeight = FontWeight.SemiBold,
+                fontFamily = FontFamily.Serif,
+                color = Color.White,
+                modifier = Modifier
+                    .padding(start = 10.dp, top = 10.dp, bottom = 15.dp)
+            )
+        }
+    }
+}
+
+@Composable
+private fun Overview(
+    modifier: Modifier,
+    movieDetailsViewState: MovieDetailsViewState,
+) {
+    Column(
+        modifier = modifier
+    ) {
+        Text(
+            text = "Overview",
+            fontSize = 17.sp,
+            fontWeight = FontWeight.SemiBold,
+            fontFamily = FontFamily.Serif,
+            color = Color.Black,
+            modifier = Modifier
+                .padding(start = 10.dp, top = 5.dp)
+        )
+        Text(
+            text = movieDetailsViewState.overview,
+            fontSize = 12.sp,
+            fontWeight = FontWeight.SemiBold,
+            fontFamily = FontFamily.Serif,
+            color = Color.Black,
+            modifier = Modifier
+                .padding(start = 10.dp, top = 5.dp)
+        )
+    }
+}
+
+@Composable
+private fun Crew(
+    modifier: Modifier,
+    movieDetailsViewState: MovieDetailsViewState,
+) {
+    LazyVerticalGrid(
```

Comment:
It probably wasn't clear in homework assignment, but here you can use non scrollable grid with fixed number of crew members, e.g. 6.